.github/workflows/functional-test.yaml

```diff
@@ -694,14 +696,38 @@ jobs:
     runs-on: ubuntu-latest
     if: failure() && github.event_name == 'schedule' && github.repository == 'radius-project/radius'
     steps:
+      - name: Count recently failed tests
+        id: count_failures
+        uses: actions/github-script@v7
+        with:
+          script: |
+            response = await github.rest.actions.listWorkflowRuns({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              workflow_id: 'functional-test.yaml',
```

Comment:
@ytimocin workflow_id will query`functional-test.yaml` workflow.

Reply:
No it should always return more than 2. The API returns workflow runs data in chronical order. L713-L719 counts only consecutive failures before the current run. the initial value of failureCount might make you confused. I can update it to 0.